src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -207,6 +207,7 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     bool OnSnapHotkeyBasedOnZoneNumber(HWND window, DWORD vkCode) noexcept;
     bool OnSnapHotkeyBasedOnPosition(HWND window, DWORD vkCode) noexcept;
     bool OnSnapHotkey(DWORD vkCode) noexcept;
+    bool ExecuteDirectedSnapHotkey(HWND window, DWORD vkCode, bool cycle, winrt::com_ptr<IZoneWindow> zoneWindow) noexcept;

```

Comment:
nit nit: 'ProcessDirectedSnapHotkey' might be better name

Reply:
Agreed